str/gangstr_runner.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+"""
+Example of running Batch script with analysis-runner.
+"""
+import os
+import hailtop.batch as hb
+import click
+
+DATASET = os.getenv('DATASET')
+HAIL_BUCKET = os.getenv('HAIL_BUCKET')
+OUTPUT_SUFFIX = os.getenv('OUTPUT')
+BILLING_PROJECT = os.getenv('HAIL_BILLING_PROJECT')
+ACCESS_LEVEL = os.getenv('ACCESS_LEVEL')
+
+REF_FASTA = 'gs://cpg-reference/hg38/v1/Homo_sapiens_assembly38.fasta'
+SAMTOOLS_IMAGE = 'australia-southeast1-docker.pkg.dev/cpg-common/images/samtools:v0'
+GANGSTR_IMAGE = "australia-southeast1-docker.pkg.dev/cpg-common/images/gangstr:v2.5"
+regions = 'gs://cpg-fewgenomes-test/hoptan-str/gangstr_catalog_with_offtarget.bed'
+
+
+@click.command()
+@click.argument('cram_path')
+@click.argument('region')
+def main(cram_path: str, region: str):  # pylint: disable=missing-function-docstring
```

Comment:
Perform the same treatment as the cram:
```suggestion
@click.argument('region_path')
def main(cram_path: str, region_path: str):  # pylint: disable=missing-function-docstring
```

Reply:
Hi Michael, does "region" refer to the second input argument? EG "chr21:1-1000" in analysis-runner \
  --access-level test \
  --dataset fewgenomes \
  --description "Run Batch" \
  --output-dir "$(whoami)-test-batch" \
  batch.py \
  gs://cpg-fewgenomes-test/analysis-runner-test/batch/input.cram \
  chr21:1-10000